src/lib/LocalTree.js

```diff
@@ -0,0 +1,265 @@
+import browser from './browser-api'
+import Bookmark from './Bookmark'
```

Comment:
Bookmark no longer exists in the src tree.  Do an import of Tree to get Bookmark.